### ISSUE TITLE
Dropdown: Default select is not easily defeated in Firefox 34+ and IE 11+

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/design-system#69]
We've discovered that the U.S. Standard dropdown is not handling hiding the default appearance of the select menu handle in certain versions of Firefox and IE.

**Comment by humancompanion** [comment]
From @raywilliams:

For now, I’ve found that I can add

select::-ms-expand {
    display: none;
}
 
to the CSS and it removes the select arrow in IE. To get IE to display the chevron, I’ve commented out the declaration
 
background-image: url(../images/png/arrow-down.svg);
 
from the select rule set. That doesn’t take care of it long term since you want to use the SVG, but it does make it work for right now. A screenshot of how it looks in IE on my local machine is below.

cid:image001.png@01D185A9.47D5B490

**Comment by humancompanion** [comment]
Ah ha. Turns out this is fixed in Firefox v35:
https://developer.mozilla.org/en-US/Firefox/Releases/35
https://bugzilla.mozilla.org/show_bug.cgi?id=649849

Annoyingly, the IT folks had me on version 34 when the latest version is 45! I just updated and sure enough it is fixed. We get less than 1% of all Firefox users (which are around 7% of our traffic) who are still stuck on a version less than 35.

I've applied the changes Ray mentioned. I think our work is done here.
